src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -240,6 +243,7 @@ export class DashboardComponent implements OnInit, OnDestroy {
   }
 
   public checkedGraph(event) {
+    this.isAlive = false;
```

Comment:
why? please add a comment here; and also with isAlive declaration (purpose comment, as variable name is not self explantaory)

Reply:
This change is not required!